<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>425</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Поле</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Maze" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>128</width>
      <height>207</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buttonAStarSearch">
       <property name="text">
        <string>Algorithm A*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLiSearch">
       <property name="text">
        <string>Algorithm Li</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClearPath">
       <property name="text">
        <string>Clear path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClearWalls">
       <property name="text">
        <string>Clear walls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClearAll">
       <property name="text">
        <string>Clear all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAStar">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelLi">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Maze</class>
   <extends>QWidget</extends>
   <header location="global">maze.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
